### DISCUSSION TITLE
建议对调软件设置中重置按钮和刷新按钮的位置

### DISCUSSION THREAD

**Discussion by xmbai9507** [shuax/MouseInc.Settings#289] [Ideas]
![image](https://user-images.githubusercontent.com/65625632/173176759-8b12b618-067f-4a34-9bff-371f606e27fc.png)

**Comment by shuax** [comment]
本来刷新F5就行，很多人不会我才加的刷新按钮。

**Comment by xmbai9507** [comment]
> 很多人不会我才加的刷新按钮

加了当然挺好, 提高了易用性, 但如果能换一下加的位置那就更好了.
建议考虑要么放到"保存"左边, 要么放到"语言"右边, 要么干脆把"保存"和"刷新"集成为一个功能按钮"保存并刷新".

**Comment by gbsix** [comment]
还是有一定道理的
